_posts/2025-03-13-CPANSec-Retrospective-2024.md

```diff
@@ -0,0 +1,152 @@
+
+This effort resulted in creating a [template](https://security.metacpan.org/docs/guides/security-policy-for-authors.html#examples), providing [guidelines](https://security.metacpan.org/docs/guides/security-policy-for-authors.html) and, last but not least, communicating about the initiative in various channels.
+
+We owe this successful initiative in particular to [RRWO](https://metacpan.org/author/RRWO).
+
+## Study on Random Number generation for Security
+Generating random numbers can help in a wide variety of tasks.
+All of them don't have the same level of criticality.
+In particular, random generated data used in a security context requires extra care.
+
+CPANSec contributor [RRWO](https://metacpan.org/author/RRWO) studied and reviewed CPAN modules for this use, and shared this in [Random Data For Security Use](https://security.metacpan.org/docs/guides/random-data-for-security.html).
+
+## Working on "TLS in core"
+A vanilla install of Perl (distribution) does not provide HTTPS capabilities, and this is annoying.
+That's an important (and difficult) topic that is addressed, across core developers and CPANSec.
+
+Things started to go into motion in 2024 to make it happen. In particular thanks to [LEONT](https://metacpan.org/author/LEONT).
+
+[Discussions](https://www.nntp.perl.org/group/perl.perl5.porters/2024/11/msg269050.html), with the Perl Steering Council or core contributor. A [plan](https://www.nntp.perl.org/group/perl.perl5.porters/2024/11/msg269202.html) was built, with some preliminary work happening.
+
+[LEONT](https://metacpan.org/author/LEONT) created the new module [Crypt::Bear](https://metacpan.org/dist/Crypt-Bear) to wrap the TLS library BearSSL.
+
+## SBOM
+CPANSec attended a billion of meetings, cross connected with several other organization working on the topic.
+
+One of the outcome of this huge effort is available as a document [Roles and metadata in open source supply-chains](https://security.metacpan.org/docs/supplychain-sbom.html) but it goes well beyond that.
+
+[SJN](https://metacpan.org/author/SJN) is leading this topic.
+
+## Adopting security related modules
+CPANSec volunteers adopted security related modules:
+- [Crypt::DSA](https://metacpan.org/dist/Crypt-DSA) adopted by [TIMLEGGE](https://metacpan.org/author/TIMLEGGE)
+- [Crypt::DES_EDE3](https://metacpan.org/dist/Crypt-DES_EDE3) adopted by [TIMLEGGE](https://metacpan.org/author/TIMLEGGE)
+- [Crypt::OpenPGP](https://metacpan.org/dist/Crypt-OpenPGP) adopted by [TIMLEGGE](https://metacpan.org/author/TIMLEGGE)
+- [Crypt::OpenSSL::AES](https://metacpan.org/dist/Crypt-OpenSSL-AES) adopted by [TIMLEGGE](https://metacpan.org/author/TIMLEGGE)
+- [Crypt::OpenSSL::Blowfish](https://metacpan.org/dist/Crypt-OpenSSL-Blowfish) adopted by [TIMLEGGE](https://metacpan.org/author/TIMLEGGE)
+- [Crypt::OpenSSL::ConfiguredAPI](https://metacpan.org/dist/Crypt-OpenSSL-ConfiguredAPI) adopted by [TIMLEGGE](https://metacpan.org/author/TIMLEGGE)
```

Comment:
I wrote this one so we can remove it from the adopt list

Reply:
OK thank you removed